source/aggregation-pipeline-builder.txt

```diff
@@ -68,13 +68,53 @@ Limitations
   or ``$searchMeta`` stages on a :ref:`time series collection
   <manual-timeseries-landing>`.
 
+
 .. _create-agg-pipeline:
 
-Create an Aggregation Pipeline
-------------------------------
+Create an Aggregation Pipeline : Stage View Mode
```

Comment:
[suggestion]

This page already has a lot of headings (probably could use a refactor to break tasks into separate pages, as part of a separate ticket). Consider nesting the Stage View Mode and Text View Mode headings under a single top-level "Create an Aggregation Pipeline" heading.

As in:

```
Create an Aggregation Pipeline
------------------------------

<some kind of introductory text about creating a pipeline>

h2. Stage View Mode
~~~~~~~~~~~~~~~~~~~

...

h2. Text View Mode
~~~~~~~~~~~~~~~~

...